ang/civicase/ActivityActions.js

```diff
@@ -0,0 +1,46 @@
+(function ($, _, angular) {
+  var module = angular.module('civicase');
+
+  module.directive('civicaseActivityActions', function () {
+    return {
+      scope: {
+        selectedActivities: '='
+      },
+      controller: civicaseActivityActionsController,
+      templateUrl: '~/civicase/ActivityActions.html',
+      restrict: 'A'
+    };
+
+    function civicaseActivityActionsController ($scope, crmApi) {
+      var ts = $scope.ts = CRM.ts('civicase');
```

Comment:
What is the use of storing `CRM.ts('civicase')` inside `$scope.ts`. 
And we are then let's use it inside template 
```html
<li>
  <a href="" ng-click="deleteActivity(selectedActivities)">{{ ts('Delete') }}</a>
</li>
```

- In future, it will be great if we can create a filter out of this and use it directly, as this has been used at many places in the app.
Something like
```js
  module.filter('civicaseTranslate', function () {
    return civicaseTranslateFilter;
  });

  /**
   * Returns the translated text
   *
   * @param {String} text to be translated
   * @return {String} translated string
   */
  function civicaseTranslateFilter (text) {
    return CRM.ts('civicase')(text);
  }

```

Reply:
Its required inside the HTML, which was missed before. Added. Thanks for noticing. For the filter, yes lets consider this after release.